specification/resources/databases/patch_database_config.yml

```
@@ -0,0 +1,47 @@
+operationId: patch_database_config
+
+summary: Change configuration values
+
+description: |
+  To update configuration for an existing database cluster, send a PATCH request to
+  `/v2/databases/$DATABASE_ID/config`.
+
+  **Note:** Database configuration is not supported for Redis and Postgres clusters.
+
+tags:
+  - Databases
+
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        allOf:
+          - $ref: 'models/database_config.yml'
+          - type: object
```

I believe that the `allOf` statement is unnecessary here, since you can use the reference directly:
```suggestion
  content:
    application/json:
      schema:
        $ref: ''models/database_config.yml''
```

---

I think this is meant to be `oneOf`, I know the plan for this API call is for it to accept different payloads depending on the engine type. I do think that the `- type: object` needs to be removed though.